src/rules/orderedImportsRule.ts

```diff
@@ -127,64 +170,105 @@ export class Rule extends Lint.Rules.AbstractRule {
 type Transform = (x: string) => string;
 const TRANSFORMS = new Map<string, Transform>([
     ["any", () => ""],
-    ["case-insensitive", (x) => x.toLowerCase()],
+    ["case-insensitive", x => x.toLowerCase()],
     ["lowercase-first", flipCase],
-    ["lowercase-last", (x) => x],
-    ["full", (x) => x],
-    ["basename", (x) => {
-        if (!ts.isExternalModuleNameRelative(x)) {
-            return x;
-        }
+    ["lowercase-last", x => x],
+    ["full", x => x],
+    [
+        "basename",
+        x => {
+            if (!ts.isExternalModuleNameRelative(x)) {
+                return x;
+            }
 
-        const splitIndex = x.lastIndexOf("/");
-        if (splitIndex === -1) {
-            return x;
+            const splitIndex = x.lastIndexOf("/");
+            if (splitIndex === -1) {
+                return x;
+            }
+            return x.substr(splitIndex + 1);
         }
-        return x.substr(splitIndex + 1);
-    }],
+    ]
 ]);
 
 enum ImportType {
     LIBRARY_IMPORT = 1,
     PARENT_DIRECTORY_IMPORT = 2, // starts with "../"
-    CURRENT_DIRECTORY_IMPORT = 3, // starts with "./"
+    CURRENT_DIRECTORY_IMPORT = 3 // starts with "./"
 }
 
 interface Options {
     groupedImports: boolean;
     importSourcesOrderTransform: Transform;
-    moduleSourcePath: Transform;
+    moduleSourceTransform: Transform;
```

Comment:
😕 changing an option name is often a breaking change... is this exposed to users?

Reply:
looks like it is not exposed, just confirming.

Reply:
Correct.  This is an internal options object.  I only changed the name to bring parallelism to the "Transform" properties in the interface.